docs/JSON-RPC-API/Authentication.md

```diff
@@ -0,0 +1,88 @@
+# Authentication and Authorization 
+
+Authentication identifies a user based on a username and password. Authorization verifies whether the user has
+access to the JSON-RPC method they are requesting.  
+
+Pantheon uses [JWT tokens](https://jwt.io/introduction/) to authenticate JSON-RPC API requests. 
+
+!!! important 
+    Ensure all API requests are made over HTTPS.  
+
+## Credentials File 
+
+The credentials file is a  `toml` file defining user details and the JSON-RPC methods to which they have access. 
+
+!!! example "Example Credentials File"
+    ```toml
+    [Users.username1]
+    password = "$2a$10$l3GA7K8g6rJ/Yv.YFSygCuI9byngpEzxgWS9qEg5emYDZomQW7fGC"
+    permissions=["net:*","eth:blockNumber"]
+    
+    [Users.username2]
+    password = "$2b$10$6sHt1J0MVUGIoNKvJiK33uaZzUwNmMmJlaVLkIwinkPiS1UBnAnF2"
+    permissions=["net:version","admin:*"]
+    ```
+
+Each user requiring JSON-RPC access is listed with: 
+
```

Comment:
for completeness should we also include username?

Reply:
Updated